kayenta-objectstore-configbin/src/main/java/com/netflix/kayenta/configbin/storage/ConfigBinStorageService.java

```diff
@@ -87,7 +89,7 @@ public <T> T loadObject(String accountName, ObjectType objectType, String object
                       () -> remoteService.get(ownerApp, configType, objectKey),
                       MAX_RETRIES,
                       RETRY_BACKOFF));
-    } catch (RetrofitError e) {
+    } catch (SpinnakerServerException e) {
       throw new NotFoundException("No such object named " + objectKey);
```

Comment:
Proabably a followup PR, but is it really appropriate to throw a NotFoundException in all cases here?  I would have though it only applies for a SpinnakerHttpException with a 404 response code...but in that case the only reason to catch and re-throw could be to add a more descriptive error message
```
throw e.newInstance("No such object named " + objectKey);
```
might cover it.